index.bs

```diff
@@ -1827,49 +1827,93 @@ To <dfn for="WebTransportError">set up</dfn> a {{WebTransportError}} |error| wit
 *This section is non-normative.*
 
 This section describes the [[QUIC]] protocol behavior of methods defined
-in this specification, utilizing [[WEB-TRANSPORT-HTTP3]].
+in this specification, utilizing [[WEB-TRANSPORT-HTTP3]]. Cause and effect may
+not be immediate due to buffering.
 
   <table class="data">
     <colgroup class="header"><col></colgroup>
     <colgroup><col></colgroup>
     <thead>
       <tr>
-        <th>Method</th>
+        <th>API Method</th>
         <th>QUIC Protocol Action</th>
       </tr>
     </thead>
     <tbody>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.{{WritableStream/abort}}(errorCode)</td>
-        <td>send RESET_STREAM with errorCode</td>
+        <td>[=stream/Reset|sends RESET_STREAM=] with errorCode</td>
```

Comment:
See below regarding the appropriate actor

Reply:
I'll fix the other one, but isn't this link correct (to reset)? The wording is unchanged. @martinthomson would you prefer different wording?

Reply:
OK, different context.  This one is fine.